### PR TITLE
Add s03e01 normal themes

### PR DESCRIPTION
**Pull request by IsaackMwenda**
Thematic document: https://docs.google.com/spreadsheets/d/1CRqhilcysa5exbTlcJEO-kAYgXMTCB4hCLIpPnV1hEc/edit#gid=1816784909

### CODE REVIEW

File: code_schemes/kakuma_s03e01_reasons.json [hunk 1]
```diff
@@ -3,7 +3,70 @@
...
+      "CodeType": "Normal",
+      "DisplayText": "provide learning resources",
+      "NumericValue": 3,
+      "StringValue": "provide_learning_resources",
+      "VisibleInCoda": true
+    },
+    {
+      "CodeID": "code-20a33b45",
+      "CodeType": "Normal",
+      "DisplayText": "provide advise",
```
Comment (as2388) [inline code_schemes/kakuma_s03e01_reasons.json]:
```suggestion
      "DisplayText": "provide advice",
```

File: code_schemes/kakuma_s03e01_reasons.json [hunk 2]
```diff
@@ -3,7 +3,70 @@
   "Name": "KAKUMA S03E01 Reasons",
   "Version": "0.0.0.2",
   "Codes": [
-
+    {
+      "CodeID": "code-a145bde3",
+      "CodeType": "Normal",
+      "DisplayText": "extra tuition",
```
Comment (as2388) [inline code_schemes/kakuma_s03e01_reasons.json]: This is "Extra Tutoring/ Tuition" in the spreadsheet, not sure if that matters

**Review by as2388 — APPROVED** [review]

File: code_schemes/kakuma_s03e01_reasons.json (hunk 1 above)
Comment (IsaackMwenda) [reply code_schemes/kakuma_s03e01_reasons.json]: updated thanks.

File: code_schemes/kakuma_s03e01_reasons.json (hunk 2 above)
Comment (IsaackMwenda) [reply code_schemes/kakuma_s03e01_reasons.json]: Was wondering whether to keep it too as they seem the same to me. Have added it back just in case.

**Pull request merged**